docs/source/getting_started/installation.rst

```diff
@@ -75,113 +79,120 @@ These docker images are used for CI and testing only, and they are not intended
 
 Latest code can contain bugs and may not be stable. Please use it with caution.
 
-Build from source (without compilation)
----------------------------------------
+Build from source without compilation
+======================================
 
-If you want to develop vLLM, and you only need to change the Python code, you can build vLLM without compilation.
+If you only need to change Python code, you can simply build vLLM without compilation.
 
-The first step is to follow the previous instructions to install the latest vLLM wheel:
+The first step is to install the latest vLLM wheel:
 
 .. code-block:: console
 
-    $ pip install https://vllm-wheels.s3.us-west-2.amazonaws.com/nightly/vllm-1.0.0.dev-cp38-abi3-manylinux1_x86_64.whl
+    pip install https://vllm-wheels.s3.us-west-2.amazonaws.com/nightly/vllm-1.0.0.dev-cp38-abi3-manylinux1_x86_64.whl
 
-After verifying that the installation is successful, we have a script for you to copy and link directories, so that you can edit the Python code directly:
+You can find more information about vLLM's wheels `above <#install-the-latest-code>`_.
+
+After verifying that the installation is successful, you can use `the following script <https://github.com/vllm-project/vllm/blob/main/python_only_dev.py>`_:
 
 .. code-block:: console
 
     $ git clone https://github.com/vllm-project/vllm.git
     $ cd vllm
     $ python python_only_dev.py
 
-It will:
+The script will:
+
+* Find the installed vLLM package in the current environment.
+* Copy built files to the current directory.
+* Rename the installed vLLM package.
+* Symbolically link the current directory to the installed vLLM package.
 
-- Find the installed vLLM in the current environment.
-- Copy built files to the current directory.
-- Rename the installed vLLM
-- Symbolically link the current directory to the installed vLLM.
+Now, you can edit the Python code in the current directory, and the changes will be reflected when you run vLLM.
 
-This way, you can edit the Python code in the current directory, and the changes will be reflected in the installed vLLM.
 
-.. _build_from_source:
+.. _with_compilation:
```

Comment:
Currently, the Adding a New Model page links to `build_from_source`. Perhaps it would be better to add another layer of nesting? e.g.

- Build from source
  - Python-only build (without compilation)
  - Full build (with compilation)
    - Use an existing PyTorch installation
    - Troubleshooting